util/helm/client.go

```diff
@@ -519,10 +398,36 @@ func (c *nativeHelmChart) GetTags(chart string, noCache bool) (*TagsList, error)
 		}
 	}
 
+	tags := &TagsList{}
 	if len(data) == 0 {
 		start := time.Now()
-		var err error
-		data, err = c.getTags(chart)
+		repo, err := remote.NewRepository(tagsURL)
+		if err != nil {
+			return nil, fmt.Errorf("failed to initialize repository: %v", err)
+		}
+		tlsConf, err := newTLSConfig(c.creds)
+		if err != nil {
+			return nil, fmt.Errorf("failed setup tlsConfig: %v", err)
+		}
+		client := &http.Client{Transport: &http.Transport{
+			Proxy:           proxy.GetCallback(c.proxy),
+			TLSClientConfig: tlsConf,
+		}}
+		repo.Client = &auth.Client{
+			Client: client,
+			Cache:  auth.DefaultCache,
```

Comment:
Do we even need a cache? This would cache the auth token based upon the registry hostname, but we always want to use whatever is incoming from the repo server. 

```suggestion
			Cache:  nil,
```